x/upgrade/client/cli/tx.go

```diff
@@ -19,8 +19,8 @@ const (
 	TimeFormat = "2006-01-02T15:04:05Z"
 
 	FlagUpgradeHeight = "upgrade-height"
-	FlagUpgradeTime   = "time"
-	FlagUpgradeInfo   = "info"
+	FlagUpgradeTime   = "upgrade-time"
```

Comment:
This is good, yet why not just changing `update-height` to `height`? Considering that the command itself is named `software-upgrade`, flags `upgrade-` prefix has always sounded a bit of an unnecessary repetition IMHO

Reply:
`--height` might be conflict with other usage? when query stuff, there's usually a block height parameter.

Reply:
`--height` is a flag for query commands only